tests/testthat/testImportM.COSMIC.R

```diff
@@ -0,0 +1,137 @@
+# testImportM.COSMIC.R
+#
+#
+context("import COSMIC mutations")
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+NL <- .PlatformLineBreak()
+
+testPath <- tempdir()
+testFName <- paste("importM_COSMIC_", Sys.Date(), ".rds", sep = "")
+# ==== END SETUP AND PREPARE ===================================================
+
+
+test_that("importM.COSMIC returns error when file not found", {
+
+    expect_error(
+        importM.COSMIC(fName = "testCosmicCNA2.tsv"))
+})
+
+test_that("importM.COSMIC returns error when fNameCNA is not a valid string", {
+    expect_error(
+        importM.COSMIC(fNameCNA = 0))
+})
+
+test_that("importM.COSMIC returns error when silent is not a valid logical", {
+    expect_error(
+        importM.COSMIC(fNameCNA = "testCosmicCNA.tsv",
+                       silent = 0))
+})
+
+test_that("importM.COSMIC returns error when writeLog is not a valid logical", {
+    expect_error(
+        importM.COSMIC(fNameCNA = "testCosmicCNA.tsv",
+                       writeLog = 0))
+})
+
+test_that("importM.COSMIC returns error when writeToFile is not a valid logical", {
+    expect_error(
+        importM.COSMIC(fNameCNA = "testCosmicCNA.tsv",
+                       writeToFile = 0))
+})
+
+test_that("importM.COSMIC saves to RDS file only when writeToFile is TRUE", {
+    expect_error(
+        importM.COSMIC(fNameCNA = "testCosmicCNA.tsv",
+                       outFName = "cosmicCNA.rds",
+                       writeToFile = FALSE))
+})
+
+test_that("importM.COSMIC returns error when outFNames is not string.", {
+    expect_error(
+        importM.COSMIC(fNameCNA = "testCosmicCNA.tsv",
+                       outFName = 0,
+                       writeToFile = TRUE))
+})
+
+test_that("importM.COSMIC correctly returns rCNA from Cosmic CNA data", {
+    fName <- "testCosmicCNA.tsv"
```

Comment:
Do. Not. Write. To. Arbitrary. Files. In. This. Directory.

Use ```tempfile()``` - and remove it after you are done.